src/sailjail.c

```diff
@@ -408,6 +420,9 @@ int main(int argc, char* argv[])
         { "quiet", 'q',
           G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, jail_arg_quiet,
           "Disable all sailjail output", NULL },
+        { "trace", 't',
+          G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, jail_arg_trace,
+          "Enable libtrace and dbus proxy logging", NULL },
```

Comment:
What's libtrace?

Reply:
Firejail allows attaching "libtrace" dso to all processes executed within the sandbox (via ld.so.preload inside sandbox) -> intercepts and logs select libc functions / syscalls and return values -> can be used to gain visibility to e.g. file access failures that the sandboxed application itself is not reporting. A bit easier / lighter than using strace / similar tools.